Getting-Started/LogViewer/index-v8.md

```diff
@@ -0,0 +1,152 @@
+            foreach (var entity in results)
+            {
+                //Reads the compact JSON format stored in the 'Data' column back to a LogEvent
+                //Same as the JSON txt files does
+                var logItem = LogEventReader.ReadFromString(entity.Data);
+
+                if (count > skip + take)
+                {
+                    break;
+                }
+
+                if (count < skip)
+                {
+                    count++;
+                    continue;
+                }
+
+                if (filter.TakeLogEvent(logItem))
+                {
+                    logs.Add(logItem);
+                }
+
+                count++;
+            }
+
+            return logs;
+        }
+
+        public override IReadOnlyList<SavedLogSearch> GetSavedSearches()
+        {
+            //TODO: Fetch from Azure Table
+            return base.GetSavedSearches();
+        }
+
+        public override IReadOnlyList<SavedLogSearch> AddSavedSearch(string name, string query)
+        {
+            //TODO: Add to Azure Table
+            return base.AddSavedSearch(name, query);
+        }
+
+        public override IReadOnlyList<SavedLogSearch> DeleteSavedSearch(string name, string query)
+        {
+            //TODO: Remove from Azure Table
+            return base.DeleteSavedSearch(name, query);
+        }
+    }
+}
+```
+
+### Register implementation
+Umbraco needs to be made aware that there is a new implementation of an `ILogViewer` to register and replace the default JSON LogViewer that we ship in the core of Umbraco.
+
+```csharp
+using Umbraco.Core;
+using Umbraco.Core.Components;
+
+namespace My.Website
+{
+    [RuntimeLevel(MinLevel = RuntimeLevel.Run)]
+    public class LogViewerSavedSearches : IUserComposer
+    {
+        public void Compose(Composition composition)
+        {
+            composition.SetLogViewer<AzureTableLogViewer>();
+        }
+    }
+}
+```
+
+### Configure Umbraco to log to Azure Table Storage
+Now with the above two classes we have the plumbing in place to view logs from an Azure Table, however we are not persisting our logs into the Azure table storage account. So we need to configure the Serilog logging pipeline to store our logs into Azure table storage.
+
+* Install Serilog.Sinks.AzureTableStorage from Nuget (Latest pre-release)
+* Update `config/serilog.user.config` with credentials (so logs persist to Azure)
+
+```xml
+<add key="serilog:using:AzureTableStorage" value="Serilog.Sinks.AzureTableStorage" />
+<add key="serilog:write-to:AzureTableStorage.connectionString" value="DefaultEndpointsProtocol=https;AccountName=ACCOUNT_NAME;AccountKey=KEY;EndpointSuffix=core.windows.net" />
+<add key="serilog:write-to:AzureTableStorage.formatter" value="Serilog.Formatting.Compact.CompactJsonFormatter, Serilog.Formatting.Compact" />
+```
+
+For more in depth information about logging and how to configure it, please read the logging documentation
```

Comment:
```suggestion
For more in depth information about logging and how to configure it, please read the [logging documentation](../Code/Debugging/Logging/index-v8).
```

Reply:
Not sure why I can't use the Commit suggestion feature here - will manually make the change @jmayntzhusen